indico/MaKaC/posterDesignConf.py

```diff
@@ -67,7 +67,7 @@ def getArgumentType(cls):
     getArgumentType = classmethod (getArgumentType)
 
     def getValue(cls, conf):
-        return conf.getOrgText()
```

Comment:
Ah yes, not sure why I changed this here. Probably didn't notice it was poster/badge-related